src/auto_scheduler/transform_step.h

```diff
@@ -411,6 +411,85 @@ class ReorderStep : public Step {
   TVM_DEFINE_OBJECT_REF_METHODS(ReorderStep, Step, ReorderStepNode);
 };
 
+/*! \brief Similar to SplitStepNode, but use split factor from another stepf
+ * (i.e. Follow another split step) */
+class FollowSplitStepNode : public StepNode {
+ public:
+  int iter_id;      // The id of the iter to split
+  int src_step_id;  // The index of the split step to follow in the history
+  int n_split;      // The number of split level
+
+  void WriteToRecord(dmlc::JSONWriter* writer) const final;
+
+  void ExtractSplitLengths(const Array<Step>& transform_steps,
+                           Array<Optional<Integer>>* lengths) const;
+
+  Array<Iterator> ApplyToState(State* state) const;
+
+  Array<tir::IterVar> ApplyToSchedule(Array<te::Stage>* stages, StageToAxesMap* stage_to_axes,
+                                      const Array<Step>& transform_steps) const;
+
+  String PrintAsPythonAPI(Array<te::Stage>* stages, StageToAxesMap* stage_to_axes,
+                          const Array<Step>& transform_steps) const;
+  static constexpr const char* record_prefix_str = "FSP";
```

Comment:
Add blank lines in front and back of this line, keep the same as other class do.